### ISSUE TITLE
Travis build broken on example tests

### ISSUE THREAD

**Issue by janetkuo** [kubernetes/kubernetes.github.io#2210]
```console 
$ go test -v k8s.io/kubernetes.github.io/test
# k8s.io/kubernetes.github.io/test_test
test/examples_test.go:53: impossible type switch case: obj (type "k8s.io/kubernetes/pkg/runtime".Object) cannot have dynamic type *"k8s.io/kubernetes/pkg/api".ReplicationController (wrong type for GetObjectKind method)
	have GetObjectKind() "k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/runtime/schema".ObjectKind
	want GetObjectKind() "k8s.io/kubernetes/pkg/runtime/schema".ObjectKind
test/examples_test.go:57: cannot use "k8s.io/kubernetes/pkg/api/validation".ValidateReplicationController(t) (type "k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/validation/field".ErrorList) as type "k8s.io/kubernetes/pkg/util/validation/field".ErrorList in assignment
test/examples_test.go:58: impossible type switch case: obj (type "k8s.io/kubernetes/pkg/runtime".Object) cannot have dynamic type *"k8s.io/kubernetes/pkg/api".ReplicationControllerList (wrong type for GetObjectKind method)
	have GetObjectKind() "k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/runtime/schema".ObjectKind
	want GetObjectKind() "k8s.io/kubernetes/pkg/runtime/schema".ObjectKind
test/examples_test.go:60: cannot use &t.Items[i] (type *"k8s.io/kubernetes/pkg/api".ReplicationController) as type "k8s.io/kubernetes/pkg/runtime".Object in argument to validateObject:
	*"k8s.io/kubernetes/pkg/api".ReplicationController does not implement "k8s.io/kubernetes/pkg/runtime".Object (wrong type for GetObjectKind method)
		have GetObjectKind() "k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/runtime/schema".ObjectKind
		want GetObjectKind() "k8s.io/kubernetes/pkg/runtime/schema".ObjectKind
test/examples_test.go:62: impossible type switch case: obj (type "k8s.io/kubernetes/pkg/runtime".Object) cannot have dynamic type *"k8s.io/kubernetes/pkg/api".Service (wrong type for GetObjectKind method)
	have GetObjectKind() "k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/runtime/schema".ObjectKind
	want GetObjectKind() "k8s.io/kubernetes/pkg/runtime/schema".ObjectKind
test/examples_test.go:66: cannot use "k8s.io/kubernetes/pkg/api/validation".ValidateService(t) (type "k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/validation/field".ErrorList) as type "k8s.io/kubernetes/pkg/util/validation/field".ErrorList in assignment
test/examples_test.go:67: impossible type switch case: obj (type "k8s.io/kubernetes/pkg/runtime".Object) cannot have dynamic type *"k8s.io/kubernetes/pkg/api".ServiceList (wrong type for GetObjectKind method)
	have GetObjectKind() "k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/runtime/schema".ObjectKind
	want GetObjectKind() "k8s.io/kubernetes/pkg/runtime/schema".ObjectKind
test/examples_test.go:69: cannot use &t.Items[i] (type *"k8s.io/kubernetes/pkg/api".Service) as type "k8s.io/kubernetes/pkg/runtime".Object in argument to validateObject:
	*"k8s.io/kubernetes/pkg/api".Service does not implement "k8s.io/kubernetes/pkg/runtime".Object (wrong type for GetObjectKind method)
		have GetObjectKind() "k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/runtime/schema".ObjectKind
		want GetObjectKind() "k8s.io/kubernetes/pkg/runtime/schema".ObjectKind
test/examples_test.go:71: impossible type switch case: obj (type "k8s.io/kubernetes/pkg/runtime".Object) cannot have dynamic type *"k8s.io/kubernetes/pkg/api".Pod (wrong type for GetObjectKind method)
	have GetObjectKind() "k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/runtime/schema".ObjectKind
	want GetObjectKind() "k8s.io/kubernetes/pkg/runtime/schema".ObjectKind
test/examples_test.go:75: cannot use "k8s.io/kubernetes/pkg/api/validation".ValidatePod(t) (type "k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/validation/field".ErrorList) as type "k8s.io/kubernetes/pkg/util/validation/field".ErrorList in assignment
test/examples_test.go:75: too many errors
FAIL	k8s.io/kubernetes.github.io/test [build failed]
The command "go test -v k8s.io/kubernetes.github.io/test" exited with 2.
```

I believe it's caused by changes made in https://github.com/kubernetes/kubernetes/pull/39475 (which just got merged), when packages like `k8s.io/kubernetes/pkg/runtime` are moved to `k8s.io/apimachinery/pkg/runtime`. We need to update examples_test.go to import those packages instead 

@chenopis @devin-donnelly

**Comment by chenopis** [comment]
@janetkuo thanks for looking into this

**Comment by steveperry-53** [comment]
@jeffmendoza

**Comment by janetkuo** [comment]
Should be fixed by #2234
